capitalize_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] We should use `capitalize`, instead of `upcase`, since we only want the first letter in the name to capital letter and the rest in lower case